res_syntax/tests/ppx/react/expected/aliasProps.res.txt

```diff
@@ -1,10 +1,7 @@
 @@jsxConfig({version: 4, mode: "automatic"})
 
 module C0 = {
-  type props<'priority, 'text> = {
-    priority: 'priority,
-    text?: 'text,
-  }
+  type props<'priority, 'text> = {priority: 'priority, text?: 'text}
```

Comment:
Wonder why this PR change leads to remove the trailing comma.

Reply:
Probably formatted differently now because of the loc changes.